RadixWallet/Features/SecurityCenterFeature/SecurityShields/ShieldSetupOnboarding/ShieldSetupOnboarding+View.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+// MARK: - ShieldSetupOnboarding.View
+extension ShieldSetupOnboarding {
+	struct View: SwiftUI.View {
+		@Perception.Bindable var store: StoreOf<ShieldSetupOnboarding>
+		@Environment(\.dismiss) var dismiss
+
+		var body: some SwiftUI.View {
+			WithPerceptionTracking {
+				coreView
+					.safeAreaInset(edge: .bottom, spacing: 0) {
+						VStack(spacing: 0) {
+							positionIndicator
+								.padding(.vertical, .small1)
+
+							let title = store.isLastStep ? L10n.ShieldsSetupOnboarding.startButtonTitle : L10n.ShieldsSetupOnboarding.nextButtonTitle
+							Button(title) {
+								store.send(.view(.nextButtonTapped))
+							}
+							.buttonStyle(.primaryRectangular)
+							.padding(.medium3)
+						}
+						.background(Color.app.background)
+					}
```

Comment:
minor, but I think it is much easier to follow if you add padding to the parent view than to each individual one:
```suggestion
					.safeAreaInset(edge: .bottom, spacing: .zero) {
						VStack(spacing: .large3) {
							positionIndicator
							let title = store.isLastStep ? L10n.ShieldsSetupOnboarding.startButtonTitle : L10n.ShieldsSetupOnboarding.nextButtonTitle
							Button(title) {
								store.send(.view(.nextButtonTapped))
							}
							.buttonStyle(.primaryRectangular)
						}
						.padding([.horizontal, .bottom], .medium3)
						.background(Color.app.background)
					}
```